muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/en.json

```diff
@@ -0,0 +1,1905 @@
+      "loadError_one_announcements": "Loading the announcement failed. Please try again.",
+      "loadError_other": "Loading the messages failed. Please try again.",
+      "loadError_other_announcements": "Loading the announcement failed. Please try again.",
+      "loadError_replies": "Loading the replies failed. Please try again.",
+      "loadError_signature": "Loading the signatures failed. Please try again.",
+      "loadError_unreadMessageCount": "Loading the unread message count failed. Please try again.",
+      "recipientsRequiredError": "The message has not recipients",
+      "removeError": "$t(common:notifications.removeError)",
+      "removeError_message": "Removing the message failed. Please try again..",
+      "removeError_announcement": "Removing the announcement failed. Please try again.",
+      "removeError_contactLog": "Removing the contact event failed. Please try again.",
+      "removeError_discussionArea": "Removing the discussion area failed. Please try again.",
+      "removeError_label": "Removing the label failed. Please try again.",
+      "removeError_reply": "Removing the reply failed. Please try again.",
+      "removeSuccess": "$t(common:notifications.removeSuccess)",
+      "removeSuccess_message": "Removing the message succeeded",
+      "removeSuccess_room": "Removing the room: {{room}} succeeded",
+      "removeSuccess_contactLog": "Removing the the contact event succeeded",
+      "restoreError": "$t(common:notifications.restoreError)",
+      "restoreError_message": "Restoring from the draft failed",
+      "restoreError_messageThread": "Restoring the thread failed",
+      "sendError": "Sending the message failed. Please try again.",
+      "sendSuccess": "Sending the message succeeded.",
+      "statusChangeError": "Message status change failed. Please try again.",
+      "updateError": "$t(common:notifications.updateError)",
+      "updateError_message": "Message editing failed. Please try again.",
+      "updateError_announcement": "Announcement editing failed. Please try again.",
+      "updateError_contactLog": "Contact log editing failed. Please try again.",
+      "updateError_label": "Updating the label failed. Please try again.",
+      "updateError_signature": "Updating the signature failed. Please try again.",
+      "updateError_discussionArea": "Updating the discussion area failed. Please try again.",
+      "updateSuccess": "$t(common:notifications.updateSuccess)",
+      "updateSuccess_message": "Updating the message failed. Please try again.",
+      "updateSuccess_announcements": "Upading the announcement failed. Please try again.",
+      "updateSuccess_contactLog": "Updating the contact event failed. Please try again.",
+      "workspacesRequiredError": "Message has no target courses"
+    },
+    "validation": {
+      "beginDate": "Announcement begin date is mandatory",
+      "caption": "$t(common:validation.caption)",
+      "caption_message": "Message caption is mandatory.",
```

Comment:
"Message title is mandatory."

Muualla näyttää olean title käytössä ja tässä tuntui olevan ainut caption termi. Olisiko parempi yhtenäistää ja käyttää titleä tässäkin.

Reply:
grepattu läpi samankaltaiset ja korjattu niissäkin